tests/behaviour/typeql/typeql_steps.py

```diff
@@ -21,6 +21,8 @@
 import re
 from collections import defaultdict
 
+import os
+import time
```

Comment:
Can remove this since we moved the step away.

Reply:
How come this comment was not addressed?